### PR TITLE
Remove folder wow moved to metal3-clusterapi-docs

### PR DESCRIPTION
**Pull request by mboukhalfa**

### CODE REVIEW

**Comment by tuminoid** [comment]
/hold
https://github.com/Nordix/metal3-clusterapi-docs/pull/96 needs to merge first.

File: wow/fullstack-build.md
Comment (mboukhalfa) [inline wow/fullstack-build.md:1]: /hold 
this need to be moved after merging https://github.com/metal3-io/project-infra/pull/629

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Review by tuminoid — APPROVED** [review]
/approve

**Comment by mboukhalfa** [comment]
/hold cancel 
PR merged

**Pull request merged**